.github/workflows/action.yml

```diff
@@ -40,6 +40,12 @@ jobs:
           node-version: '14'
       - run: npm ci
       - run: npm test
+      - name: .tool-versions test
+        id: setup-beam
+        uses: ./
+        with:
+          version-file: __tests__/.tool-versions
+          version-type: strict
```

Comment:
I understand the rationale but I think it would be more convenient if we didn't have to specify version-type if version-file is set. (we'd fail if it is set to something else than strict though.) Not a strong opinion.

Reply:
Are there actually cases where people put their `.tool-versions` deeper in their project? I think convention over configuration would indeed be better here, and then you provide a boolean. `versions-from-file: true` or something like that.

Reply:
`version-type` is `loose` by default, so not specifying it is the same as choosing non-strict. I guess validating, in that case, unless it is `strict`, would always fail (?).

There is, however, a [note on the README for this](https://github.com/erlef/setup-beam/pull/159/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R97), so it'll fail at most once on your first run, where you include the new option. This seems safe.

@starbelly, as for a new option alongside a new option, I don't understand the rationale.

> Are there actually cases where people put their .tool-versions deeper in their project?

It wouldn't matter, because the action reads a path, so you can put it wherever you want. The fact I named it like that, taking into account the issue that gave origin to this pull request, is that we might support more conventions in the future. If we think we won't, we should just change the option name to `parse-tool-versions` and use a default as you propose.

Reply:
@paulo-ferraz-oliveira Eh, it's not really a big deal IMO. I think the way you have it is more flexible, what  I just suggested would result in almost the same amount of typing 😁